ckanext/ytp/comments/email_notifications.py

```diff
@@ -151,7 +152,7 @@ def get_admins(owner_org, user, content_type, content_item_id):
     return users
 
 
-def notify_admins(owner_org, user, template, content_type, content_item_id, comment_id):
+def notify_admins(owner_org, user, template, content_type, content_item_id, comment_id, dataset_title, comment):
```

Comment:
can we please change `dataset_title` to `content_item_title` to make it generic?

Reply:
Have changed the variable name to be more generic 'content_item_title'
Also removed the unused method notify_admins